packages/appstore/src/modules/trading-hub/index.tsx

```diff
@@ -1,7 +1,9 @@
 import * as React from 'react';
+import Onboarding from 'Components/onboarding';
+import { trading_hub_contents } from '../../constants/trading-hub-content';
```

Comment:
```suggestion
import { trading_hub_contents } from 'Constants/trading-hub-content';
```

Reply:
Made all of the changes according to the suggestions.